### ISSUE TITLE
Verbose debugging - post data in my logs

### ISSUE THREAD

**Issue by benjibee** [joppuyo/acf-image-aspect-ratio-crop#34]
Going through a client's PHP error logs I kept seeing large dumps of data that seemed to be entire post objects printed to `error_log()` but couldn't figure out if I left some code in a plugin or theme I developed.

I eventually came across these calls to the `debug()` method adding the the `save_post` ACF action:

```php
add_action(
    'acf/save_post',
    function ($post_id) {
        $this->debug('post_id');
        $this->debug($post_id);
        $this->debug('POST');
        $this->debug($_POST);
       // [...]
    }
);
```

And seeing the debug function is defined as:

```php
function debug($message)
{
    if (defined('WP_DEBUG') && WP_DEBUG === true) {
        error_log(print_r($message, true));
    }
}
```

I can see how this would easily be an issue. I'm not sure why anyone using this plugin would want their PHP error logs filled with debug info of this verbosity but using the `WP_DEBUG` constant seems like a pretty bad idea to me. Perhaps an option to enable verbose debugging in the plugin would be more useful?

**Comment by joppuyo** [comment]
I agree, these lines were left for my debugging purposes. I don't think they are needed in the released plugin.

I'll get rid of these in the next version.

**Comment by joppuyo** [comment]
This has been fixed in v4.1.0.
